plugins/smart_proxy_reports/reports.rb

```diff
@@ -0,0 +1 @@
+gem 'smart_proxy_reports', '1.0.0'
```

Comment:
you also need to update https://github.com/theforeman/foreman-packaging/blob/11b2a38923980d98807fdf33b3659ea1515a41a4/plugins/smart_proxy_reports/install#L1
to use `debian/reports.rb` instead of [`bundler.d/reports.rb`](https://github.com/theforeman/smart_proxy_reports/blob/develop/bundler.d/reports.rb) as source

the target is correct